modules/shared/web/client/src/main/scala/web/client/table/package.scala

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package web.client
+
+import cats.Eq
+import cats.data.NonEmptyList
+import cats.implicits._
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.raw.JsNumber
+import japgolly.scalajs.react.Callback
+import org.scalajs.dom.MouseEvent
+import scala.scalajs.js
+import monocle.Lens
+import mouse.boolean._
+import react.virtualized._
+import react.draggable._
+import web.client.utils._
+
+package table {
+
+  sealed trait UserModified extends Product with Serializable
+  case object IsModified  extends UserModified
+  case object NotModified extends UserModified
+
+  object UserModified {
+    implicit val eq: Eq[UserModified] = Eq.fromUniversalEquals
+
+    def fromBool(b: Boolean): UserModified = b.fold(IsModified, NotModified)
```

Comment:
I prefer `if (b) IsModified else NotModified` because it's so easy to swap the arguments to `fold`.